packages/transaction-controller/src/TransactionController.ts

```diff
@@ -957,6 +971,53 @@ export class TransactionController extends BaseController<
     }
   }
 
+  /**
+   * Appends new sendFlowHistoryToAdd to the transaction with the given transactionID
```

Comment:
Maybe `Appends new send flow history to a transaction.` ?

I think the parameter and return description cover the more technical aspects.